components/widgets/inputs/users_emails_input.tsx

```diff
@@ -300,8 +375,18 @@ export default class UsersEmailsInput extends React.PureComponent {
                     defaultOptions={false}
                     defaultMenuIsOpen={false}
                     openMenuOnClick={false}
-                    loadingMessage={this.loadingMessage}
-                    onInputChange={this.handleInputChange}
+                    loadingMessage={
+                        // I don't think I can retain the same behavior here without refactoring
+                        // this function or some how changing the accepted types of `loadingMessage
+                        // property of `AsyncSelect` 
+                        this.loadingMessage
+                    }
+                    onInputChange={
+                        // I have fiddled with the type signature for this function a lot, the way it
```

Comment:
Can you post the error you're getting?

Reply:
```
Type '() => JSX.Element' is not assignable to type '(obj: { inputValue: string; }) => string | null'.
  Type 'Element' is not assignable to type 'string'.ts(2322)
Select.d.ts(144, 3): The expected type comes from property 'loadingMessage' which is declared here on type 'IntrinsicAttributes & Props<AppSelectOption> & AsyncProps<AppSelectOption> & { children?: ReactNode; }'
```

Though I think the issue is that I was going off of what vs code was telling me was an error, I don't get any of the errors I am seeing in code when I run `make test`. The errors taht I do get seem to be tests some where else in the code, not sure whats causing those.

Reply:
```
Type '(inputValue: IOptionType, action: { action: string; }) => void' is not assignable to type '(newValue: string, actionMeta: InputActionMeta) => void'.
  Types of parameters 'inputValue' and 'newValue' are incompatible.
    Type 'string' is not assignable to type 'IOptionType'.ts(2322)
Select.d.ts(173, 3): The expected type comes from property 'onInputChange' which is declared here on type 'IntrinsicAttributes & Props<AppSelectOption> & AsyncProps<AppSelectOption> & { children?: ReactNode; }'
```